Allow environment variables to be passed (#82)
huggingface/llm.nvim: README.md

`lsp.version` is used only when **llm.nvim** downloads **llm-ls** from the release page.

> `lsp.cmd_env` can be used to set environment variables to be used for llm-ls. Currently only `LLM_LOG_LEVEL` might be useful.

Reviewer: ```suggestion
`lsp.cmd_env` can be used to set environment variables for the **llm-ls** process.
```
